assets/src/edit-story/components/colorPicker/colorPicker.js

```diff
@@ -79,6 +81,7 @@ function ColorPicker({
   onChange,
   onClose,
   renderFooter,
+  changedStyle = 'background',
 }) {
```

Comment:
This component is still updating the store when the denounced callback is invoked - even though it's updating the color instantly when dragging around. Could we not completely remove the updating of global color until the color picker is closed (similarly to what I did for image edit) and only keep it in transform and locally until then? Then you can also undo color changes in a single undo step regardless of how many things you did in the color picker?

I know that's a larger task, and we can also delay that for a future ticket, but it seems like a natural extension of this one though.

Reply:
Will create a separate issue for it -- I have other priorities right now and don't want to delay the PR because of that part only, easier to follow up separately.

Reply:
Alright, no problem.